tests/unit/utils.test.ts

```diff
@@ -48,3 +52,13 @@ describe('pruneTags', () => {
     assert.deepEqual(prunedTags, []);
   });
 });
+
+describe('getCurrentBlockUnixTimestamp', () => {
+  it('should return the current block timestamp', async () => {
+    // stub arweave block request using nock
+    const arweave = Arweave.init({});
+    const minTimestamp = Date.now();
+    const timestamp = await getCurrentBlockUnixTimestamp(arweave);
+    assert.ok(timestamp >= minTimestamp);
```

Comment:
where does `nock` effect this?

Reply:
i'm not using nock atm

Reply:
will remove the comment

Reply:
[c98de0f](https://github.com/ar-io/ar-io-sdk/pull/249/commits/c98de0f11e81631cea5ebe1cac310dc61074353e)